unified-scheduler-pool/src/lib.rs

```diff
@@ -262,11 +262,11 @@ clone_trait_object!(BankingPacketHandler);
 pub struct BankingStageHelper {
     usage_queue_loader: UsageQueueLoader,
     next_task_id: AtomicUsize,
-    new_task_sender: Sender<NewTaskPayload>,
+    new_task_sender: Weak<Sender<NewTaskPayload>>,
```

Comment:
this is least invasive change to fix the circular dependency in scheduler.
We may want to re-structure the scheduler thread to not hold the entire `HandlerContext`